pkg/operator/controller/garden/garden/components.go

```diff
@@ -793,8 +853,8 @@ func getLoadBalancerServiceAnnotations(garden *operatorv1alpha1.Garden) map[stri
 	return nil
 }
 
-func (r *Reconciler) newVirtualSystem() component.DeployWaiter {
-	return virtualgardensystem.New(r.RuntimeClientSet.Client(), r.GardenNamespace, virtualgardensystem.Values{})
+func (r *Reconciler) newVirtualSystem(enableSeedAuthorizer bool) component.DeployWaiter {
```

Comment:
Do we really need this case, given the assumption that the Authorizer will eventually be enabled - before any seed joins?

Reply:
You're right, however this is only useful for the upgrade case when people deploy `gardener-operator` while already having a Gardener installation and didn't enable the seed authorizer feature before.